content/post/lapjv.md

```diff
@@ -441,8 +441,8 @@ A \\cdot \\vec{x} \\leq \\vec{b}
 \\vec{x} \\geq 0
 \\end{equation}
 We minimize the scalar product of \\(\\vec{c}\\) and \\(\\vec{x}\\) given the set
-of linear inequations depending of \\(\\vec{x}\\) and the requirement that all
```

Comment:
I suggest using `inequality` instead of `inequation`.  Both are correct, but the former is much more common than the later.

The sentence would look like this: `We minimize A and B given the set of linear inequalities depending of C...`